.github/workflows/test.yml

```
@@ -0,0 +1,27 @@
+name: Test
+on:
+  - push
+  - pull_request
```

Suggestion from @seemethere:
```suggestion
  - pull_request_target
```